### DISCUSSION TITLE
chia blockchain version 1.2.11, can't change pool, can't send chia from wallet

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#10526] [Support]
Hi,
I'm writing to check if there is something wrong with the application ver. 1.2.11. I've tried to change a pool and I've already tried few pools. In all cases it failed. "Pooling" status stays and it doesn't even show for a second "pending" status. The pool I'm trying to change from stays the same. Has anyone had similar issue? I'm asking to make myself sure is this the app or something more like issues with pool itself (I'm dropping europe.ecochia.io and want to switch to na1.pool.space or eu.pool.space). Maybe someone here may have an idea. Thanks in advance
Regards

**Comment by loppefaaret** [comment]
there is many transactions on the blockchain, and you might need to add a fee to complete you transaction to transfer your plotNFT from one pool to another
